docs/developer-docs/getting-started/cycles/cycles-ledger.mdx

```diff
@@ -0,0 +1,151 @@
+# Cycles ledger
+
+## Overview
+
+When developing and deploying canisters on the Internet Computer, developers need to have **cycles** to pay for the resources used by their canisters. The **cycles ledger** canister has been developed as an alternative to the **cycles wallet** canister for cycles management. 
+
+The biggest difference between the cycles ledger and cycles wallet is that the cycles ledger is a single global ledger canister, whereas the cycles wallet canister must be deployed separately for every developer to be used. Additionally, the cycles wallet has some downsides that the cycles ledger remedies. Some of these are:
+
+- Cycles wallets are a complex solution to associate cycles with principal IDs.
+- Cycles wallets consume cycles themselves, meaning developers need to pay cycles to store cycles.
+- Since the cycles wallet canister must be deployed separately if a cycles wallet canister ID is lost, the cycles in that cycles wallet canister are also lost. 
```

Comment:
I think the sentence needs to be rephrased. The first part of the sentence could be understood as "if I lose my wallet canister ID, then I must deploy the cycles wallet separately".
Maybe we can just drop the "Since the cycles wallet canister must be deployed separately" part?

Reply:
Removed "Since the cycles wallet canister must be deployed separately"